index.js

```diff
@@ -11,24 +11,21 @@ class BlockedMonitor {
   /**
    * Creates an instance of BlockedMonitor
    *
-   * @param {Number} delay A "warmup time" during which we tolerate blocking
-   * the loop, in milliseconds.
-   * @param {Number} threshold The maximum acceptable event loop blocking time,
-   * in milliseconds.
    */
-  constructor(delay, threshold) {
-    this.delay = delay;
-    this.threshold = threshold;
+  constructor() {
     this._intervalId = null;
     this.running = false;
   }
 
   /**
    * Starts the monitoring. The "warmup time" counts from here.
    *
-   * @returns {BlockedMonitor} the instance, for chainability.
+   * @returns {void}
    */
   start() {
+    this.delay = getIntFromEnv(process.env, 'BLOCKED_DELAY', 2000);
```

Comment:
the constructor seemed a legit place to do this, but there is no actual impact so OK